### PR TITLE
Updates with new features, updated README.md, and local server.

### PR DESCRIPTION
**Pull request by reyarqueza**

### CODE REVIEW

**Comment by captchadex** [comment]
Thanks!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "1.0.0",
+  "description": "Native HTML5 web component example.",
+  "main": "index.html",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "http-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/reyarqueza/web-component-hello-world.git"
```
Comment (reyarqueza) [inline package.json]: Oops, you can change this to your repo of course 👍

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "keywords": [
+    "web",
+    "component"
+  ],
+  "author": "Ryan Lee, Rey Arqueza",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/reyarqueza/web-component-hello-world/issues"
```
Comment (reyarqueza) [inline package.json]: same as above

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "homepage": "https://github.com/reyarqueza/web-component-hello-world#readme",
```
Comment (reyarqueza) [inline package.json]: same as above.

**Pull request merged**